README.md

```diff
@@ -40,13 +40,21 @@ pipenv run flake8
 
 Two docker-compose files are made available in this repo for standing up a local dev environment. The `full-stack.yml` file stands up ingress, kafka, yuptoo, minio, and inventory components so that the entire first bits of the platform pipeline can be tested. The `docker-compose.yml` file stands up services without yuptoo, the yuptoo can be run manaully in local for development tests.
 
-Stand Up Full Stack
+### Stand Up Full Stack
+
+🐧 For Linux Users
 
 ```sh
 cd scripts && source .env && sudo docker-compose -f full-stack.yml up
 ```
 
-Stand Up Yuptoo Manually outside the docker-compose
+🍎 For macOS Users
+```sh
+cd scripts && source .env && sudo docker-compose -f full-stack-mac.yml up
+```
+⚠️ Make sure Docker Desktop is running on macOS before executing.
```

Comment:
@hkx303 For some reason, would you mind replace this `Docker Desktop` with `Podman Desktop`?

Reply:
Sure, then I think I will remove this line directly